src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/RequestMoreTimeForResponseTest.java

```diff
@@ -26,15 +26,21 @@
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 import static uk.gov.hmcts.cmc.claimstore.controllers.utils.sampledata.SampleClaim.SUBMITTER_EMAIL;
+import static uk.gov.hmcts.cmc.claimstore.events.utils.sampledata.SampleClaimIssuedEvent.SUBMITTER_FORENAME;
+import static uk.gov.hmcts.cmc.claimstore.events.utils.sampledata.SampleClaimIssuedEvent.SUBMITTER_SURNAME;
 import static uk.gov.hmcts.cmc.claimstore.utils.VerificationModeUtils.once;
 
 public class RequestMoreTimeForResponseTest extends BaseTest {
 
     private static final String AUTH_TOKEN = "it's me!";
     private static final long DEFENDANT_ID = 100L;
     private static final LocalDate RESPONSE_DEADLINE = LocalDate.now().plusDays(10);
-    private static final UserDetails USER_DETAILS = new UserDetails(DEFENDANT_ID, "myemail@example.com");
-    private static final UserDetails OTHER_USER_DETAILS = new UserDetails(SUBMITTER_ID, "other@example.com");
+    private static final UserDetails USER_DETAILS
+        = new UserDetails(DEFENDANT_ID, "myemail@example.com", SUBMITTER_FORENAME, SUBMITTER_SURNAME);
```

Comment:
Can we use a builder with defaults set and only pick the email?

Reply:
actioned